.travis.yml

```diff
@@ -1,32 +1,48 @@
-language: python
-sudo: false
-python:
-  - 2.7
-  - 3.4
-  - 3.5
-  - 3.6
+language: generic
+
+env:
+  global:
+    - INSTALL_EDM_VERSION=1.10.0
+      PYTHONUNBUFFERED="1"
+
+matrix:
+  include:
+    - env: RUNTIME=2.7
+    - env: RUNTIME=3.5
+    - env: RUNTIME=3.6
+    - os: osx
+      env: RUNTIME=2.7
+    - os: osx
+      env: RUNTIME=3.5
+    - os: osx
+      env: RUNTIME=3.6
+  fast_finish: true
+
 branches:
   only:
     master
+
 cache:
   directories:
-    - $HOME/.cache
-    - $HOME/.ccache
+    - "~/.cache"
+
 before_install:
-  - ccache -s
-  - export PATH=/usr/lib/ccache:${PATH}
-  - pip install --upgrade pip
+  - mkdir -p "${HOME}/.cache/download"
+  - export DISPLAY=:99.0
+  - if [[ ${TRAVIS_OS_NAME} == "linux" ]] ; then sh -e /etc/init.d/xvfb start; fi
```

Comment:
It's possible that we don't need the framebuffer (though it probably doesn't do any harm).